src/screens/ProfileScreen.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useContext } from 'react';
+import { Button, View } from 'react-native';
+import { Auth } from 'aws-amplify';
+import { AppContext } from '../contexts/AppContext';
+
+export default function ProfileScreen() {
+  const { setAppState } = useContext(AppContext);
+
+  const signOut = async () => {
+    try {
+      await Auth.signOut();
+      setAppState('Auth');
+    } catch (error) {
+      console.error('Error signing out: ', error);
+    }
+  };
+
+  return (
+    <View>
+      <Button title="Sign Out" onPress={signOut} />
+    </View>
+  );
```

Comment:
I'm seeing this recurring code pattern in our codebase:
```
export default function Foo() { ...
    const Bar1 = () => { ...
        /* code block*/
    }
    const Bar2 = () => { ...
        /* code block*/
    }
```
Not to be picky, but for visual consistency/cohesiveness, perhaps we should all settle on:
```
//--- OPT 1 ---
export default function Foo() { ...
    function Bar1() { ...
        /* code block*/
    }
    function Bar2() { ...
        /* code block*/
    }
```
```
//--- OPT 2 ---
export const Foo = () => { ...
    const Bar1 = () => { ...
        /* code block*/
    }
    const Bar2 = () => { ...
        /* code block*/
    }
```

Reply:
They are [not entirely the same](https://stackoverflow.com/questions/34361379/are-arrow-functions-and-functions-equivalent-interchangeable). I think for most cases we can use fat arrow syntax but use the normal function syntax when necessary.

Reply:
my naiivete. thanks for bringing my attention to that.